src/react/src/components/InitialClaimFlow/ClaimForm/validationSchemas.js

```
@@ -1,19 +1,86 @@
 // Step 2: Contact validation.
+const getEmploymentVerificationLabel = value =>
+    EMPLOYMENT_VERIFICATION_OPTIONS.find(opt => opt.value === value)?.label;
+
+const documentEmploymentVerificationBasedLabels = new Set(
+    DOCUMENT_BASED_VERIFICATION_OPTIONS.map(getEmploymentVerificationLabel),
+);
+
+const getCompanyUrlValidationSchema = label =>
+    Yup.string().when('claimantEmploymentVerificationMethod', {
+        is: value => value === label,
+        then: schema =>
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)